.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,29 +14,40 @@ Changelog:
 ## Types of changes
 
 What types of changes does your code introduce to Zowe Explorer?
-_Put an `x` in the boxes that apply_
+_Put an `x` in all boxes that apply._
 
-- [ ] Bugfix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Enhancement (non-breaking change which adds or improves functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] Updates to Documentation or Tests (if none of the other choices apply)
+- [ ] Documentation (Markdown, README updates)
+- [ ] Other (please specify above in "Proposed changes" section)
 
 ## Checklist
 
-_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_
+_Put an `x` in all boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer._
+
+**General**
 
 - [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/zowe-explorer-vscode/wiki/Best-Practices:-Contributor-Guidance) wiki
-- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
-- [ ] PR Description is included
-- [ ] gif or screenshot is included if visual changes are made
-- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
-- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
-- [ ] I have added unit test and it is passing
-- [ ] I have added integration test and it is passing
+- [ ] All PR dependencies have been merged and published (if applicable)
+- [ ] A GIF or screenshot is included in the PR for visual changes
+- [ ] The pre-publish command has been executed:
+  - **v2 and below:** `yarn workspace vscode-extension-for-zowe vscode:prepublish`
+  - **v3:** `pnpm --filter vscode-extension-for-zowe vscode:prepublish`
+
+**Code coverage**
+
 - [ ] There is coverage for the code that I have added
-- [ ] I have tested it manually and there are no regressions found
-- [ ] I have added necessary documentation (if appropriate)
-- [ ] Any PR dependencies have been merged and published (if appropriate)
+- [ ] I have added new test cases and they are passing
+- [ ] I have manually tested the changes
+
+**Deployment**
+
+- [ ] I have added developer documentation (if applicable)
+- [ ] Documentation should be added to Zowe Docs
+  - If you're an outside contributor, please contact @anaxceron (ana.ceron@broadcom.com) to coordinate documentation.
```

Comment:
does the docs squad have a formal place like the repo to open issues to add documentation linking feature work item or something like that?  This way no personal contact info is displayed and same process will be in place if the doc contact changes in future.

Reply:
I like the idea to have some process that does not involve an individual.
Maybe opening an issue in the docs-site repo, which then will be labeled as Zowe Explorer (or Zowe clients) might work 😋

Reply:
I would also prefer to have a process outlined so that it is not specific to one individual. However, I discussed this with Ana yesterday and she said that it was okay to put her name in the template. But, if we have any other ideas that we can offer to the Doc Squad for this process, or if there is an alternative that we can put in the template, I will update this item.

Tagging for awareness @anaxceron

Reply:
I replaced Ana's contact info with a link to the #zowe-doc Slack channel in 682e419